homeassistant/components/automation/blueprints/motion_light.yaml

```diff
@@ -10,47 +10,47 @@ blueprint:
       selector:
         entity:
           filter:
-            device_class: motion
-            domain: binary_sensor
+            domain:
+              - binary_sensor
+              - group
```

Comment:
Groups of binary sensors still have the domain `binary_sensor`. It's legacy groups that have the domain `group`. I don't think we should allow those groups.